gulpfile.js

```diff
@@ -212,6 +212,7 @@ gulp.task('browser-sync', () => {
         server: {
             baseDir: './'
         },
+        cors: true,
```

Comment:
Security concern?

Reply:
This is to allow access 302 redirection; if not included when connecting to download sha256.txt file, then the broswer will reject the request
On the other hand, if the response to the redirection from github does not includes cors headers on 302 response, the browser also reject the request